ansible/roles/operator-pipeline/templates/openshift/tasks/verify-ci-results.yml

```diff
@@ -16,12 +17,21 @@ spec:
     - name: verify-ci-results
       image: registry.access.redhat.com/ubi8-minimal
       workingDir: $(workspaces.results.path)
+      env:
+        - name: PREFLIGHT_MIN_VERSION
+        - value: $(params.preflight_min_version)
       script: |
         #! /usr/bin/env bash
-        # Did the tests pass?
+
         # did preflight/ CI have the latest version?
-        # If results exists, store them in the workspace
-        # In case of failure- comment on the PR with tailored support link
+        PREFLIGHT_VERSION=$(cat PREFLIGHT_RESULTS | jq '.test_library.version' | grep .)
+        if [ "$(printf '%s\n' "$PREFLIGHT_MIN_VERSION" "$PREFLIGHT_VERSION" | sort -V | head -n1)" = "$PREFLIGHT_MIN_VERSION" ]; then
```

Comment:
```suggestion
        if [ "$(printf '%s\n' "$PREFLIGHT_MIN_VERSION" "$PREFLIGHT_VERSION" | sort -V | head -n1)" == "$PREFLIGHT_MIN_VERSION" ]; then
```

Reply:
It doesn't matter actually, still, it compares the same as ==.